File: CompositeDemo/CompositeDemo.java
import java.text.MessageFormat;
import java.util.ArrayList;

class Developer {

  String name;
  double salary;

  Developer(String name, double salary) {
    this.name = name;
    this.salary = salary;
  }
    
}

class Manager  {

  String name;
  double salary;

  Manager(String name, double salary) {
    this.name = name;
    this.salary = salary;
  }
}

public class CompositeDemo {
  public static void main(String[] args) {
    var devList = new ArrayList<Developer>();
    var mgrList = new ArrayList<Manager>();
    
    devList.add(new Developer("Brendan Eich", 100_000));
    devList.add(new Developer("James Gosling", 200_000));
    devList.add(new Developer("Guido van Rossum", 250_000));
    devList.add(new Developer("Anders Hejlsberg", 350_000));
    mgrList.add(new Manager("Dennis Ritchie", 500_000));
    mgrList.add(new Manager("Alan Kay", 500_000));

    var sum = 0;
    for (var dev : devList )
      sum += dev.salary;

    for (var mgr: mgrList)
      sum += mgr.salary;

    System.out.println(MessageFormat.format("The total company salary is {0}", sum));
  }
}

File: CompositeDemo/CompositeDemo.py
class Developer:
    def __init__(self, name: str, salary: float) -> None:
        self.name: str = name
        self.salary: float = salary

class Manager:
    def __init__(self, name: str, salary: float) -> None:
        self.name: str = name
        self.salary: float = salary

dev_list: list[Developer] = []
mgr_list: list[Manager] = []

dev_list.append(Developer("Brendan Eich", 100_000))
dev_list.append(Developer("James Gosling", 200_000))
dev_list.append(Developer("Guido van Rossum", 250_000))
dev_list.append(Developer("Anders Hejlsberg", 350_000))
mgr_list.append(Manager("Dennis Ritchie", 500_000))
mgr_list.append(Manager("Alan Kay", 500_000))

sum: float = 0
for dev in dev_list:
    sum += dev.salary

for mgr in mgr_list:
    sum += mgr.salary

print(f"The total company salary is {sum}")
